.gitignore

```
@@ -1 +1,2 @@
 node_modules/
+.DS_Store
```

```suggestion
.DS_Store

```

We need this trailing newline here otherwise there is a big red icon appearing in the web interface here for some reason.